### ISSUE TITLE
New BLAST XML format

### ISSUE THREAD

**Issue by pjotrp** [pjotrp/blastxmlparser#4]
>> The NCBI is now making a new version of the BLAST XML available for
>> testing.  Read about the changes and how to access BLAST results using
>> the new XML at ftp://ftp.ncbi.nlm.nih.gov/blast/documents/NEWXML/xml2.pdf

Anyone an interest in this?

**Comment by n8upham** [comment]
Yes for sure, I'm interested— especially for the use of taxids within the more-easily parsable XML format.  Is it already implemented anywhere?  Thanks

**Comment by pjotrp** [comment]
@n8upham send me an E-mail and we can discuss. Taxids were sorely missing, indeed.

**Comment by n8upham** [comment]
Hi there
Yes, do you know if the new XML format is (a) up and running, and (b) use-able from the UNIX executable (2.2.31+ now)?  My aim for the XML output is to be able to use biopython’s blast-based tools for parsing.  But right now the option “5” seems to still output regular XML (i.e., without the taxis, which I am keen to use)

Thanks much
—nate
